new.css

```diff
@@ -1,3 +1,7 @@
 .something {
   appearance: textfield;
 }
+
+.masker {
+  mask: url('');
```

Comment:
CSS Masks not supported by: IE (10,11), Firefox (37,38), Chrome (42,43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.3), IE Mobile (10,11)